test/.stats-app/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "license": "MIT",
   "dependencies": {
-    "next": "19.0.0-rc-6230622a1a-20240610",
+    "next": "latest",
```

Comment:
Drive-by. The version specified was not actually a Next.js version. It's overridden anyway.